### ISSUE TITLE
Youtube

### ISSUE THREAD

**Issue by itamblyn** [anki-community/website#1]
I have downloaded 6 videos related to the SDK from their youtube channel. If and when it goes down I can reupload and host from my account. I would do it now, but it's possible it will get flagged as copyrighted material (since it will be a duplicate)

**Comment by LinusS1** [comment]
Yeah, lets wait for the channel to go down, then we can deal with the videos
